spec/generator/rules/node_spec.js

```diff
@@ -74,20 +74,20 @@ describe('Azk generators Node.js rule', function() {
     h.expect(evidence).to.have.deep.property('ruleName', 'node08');
   });
 
-  it('should get latest version when version is v0.10', () => {
+  it('should get latest version when version is v0.12', () => {
     var packageJsonfilePath = '/tmp/azk-test-30501680wvr4/front/package.json';
     var packageJsonContent = [
       '{',
       '  "name": "best-practices",',
       '  "engines": {',
-      '    "node": "v0.10.0"',
+      '    "node": "v0.12.0"',
       '  },',
       '  "author": "Charlie Robbins <charlie@nodejitsu.com>"',
       '}',
     ].join('\n');
 
     var evidence = rule.getEvidence(packageJsonfilePath, packageJsonContent);
-    h.expect(evidence).to.have.deep.property('ruleName', 'node010');
+    h.expect(evidence).to.have.deep.property('ruleName', 'node012');
   });
 
   it('should get 0.11 version when version is >= v0.11', () => {
```

Comment:
it is: `should get 0.12 version when version is >= v0.11`
I added 1 more test to check ?! 0.13 ?! versions too.